templates/CMakeLists.txt.template

```diff
@@ -97,6 +97,9 @@
   set(gRPC_PROTOBUF_PROVIDER "module" CACHE STRING "Provider of protobuf library")
   set_property(CACHE gRPC_PROTOBUF_PROVIDER PROPERTY STRINGS "module" "package")
 
+  set(gRPC_PROTOBUF_PACKAGE_TYPE "" CACHE STRING "Algorithm for searching protobuf package")
```

Comment:
Do we actually need this? It looks like that currently, we first try find_package(protobuf CONFIG) and then fall back to find_package(Protobuf MODULE) if not found, so I am not sure how helpful gRPC_PROTOBUF_PACKAGE_TYPE is.